test/e2e/network/loadbalancer.go

```diff
@@ -1236,6 +1244,134 @@ var _ = common.SIGDescribe("LoadBalancers", func() {
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	ginkgo.It("should not have connectivity disruption during rolling update", func() {
+		ns := f.Namespace.Name
+		name := "test-lb-rolling-update"
+		labels := map[string]string{"name": name}
+		dsRetryPeriod := 1 * time.Second
+		dsRetryTimeout := 5 * time.Minute
+		ds := e2edaemonset.NewDaemonSet(name, e2eapps.AgnhostImage, labels, nil, nil,
+			[]v1.ContainerPort{
+				{ContainerPort: 80},
+			},
+			"netexec", "--http-port=80", "--udp-port=80",
```

Comment:
Does `netexec` continue to serve traffic even after receiving SIGTERM? I think that would be important for this test case

Reply:
Very good point!

Looking at the E2E tests tagged `[Feature:ProxyTerminatingEndpoints]`, I noticed this:
https://github.com/kubernetes/kubernetes/blob/21916415cb33e93a6ffa7f3166f9468ffaeba731/test/e2e/network/service.go#L2899-L2903

We are using [--delay-shutdown](https://github.com/kubernetes/kubernetes/blob/21916415cb33e93a6ffa7f3166f9468ffaeba731/test/images/agnhost/netexec/netexec.go#L144) flag, whose purpose is exactly this: serve traffic even after receiving `SIGTERM`.

Since the test case from here relies on `[Feature:ProxyTerminatingEndpoints]`, I'll use the same `netexec` arguments used there.

This also brings another question: should this test case be tagged with `[Feature:ProxyTerminatingEndpoints]` as well ?

Reply:
PR updated with `--delay-shutdown` flag.

Reply:
> This also brings another question: should this test case be tagged with [Feature:ProxyTerminatingEndpoints] as well ?

I think we can leave it out for now, given this tests has broader scope

Reply:
Just curious: was it the `--delay-shutdown` change that got the tests to have almost 100% success rate or was there another change needed?

Reply:
It was the `--delay-shutdown` change that got us more success rate.